rust/stub/src/part_discovery.rs

```diff
@@ -0,0 +1,19 @@
+use uefi::{Result, Handle, prelude::BootServices, proto::device_path::{DevicePath, media::{HardDrive, PartitionSignature}, DeviceType, DeviceSubType}, Guid};
```

Comment:
The code form this module is also in `efivars.rs` now. So it can be deleted now, right?

Reply:
Right, forgot it